godtools/App/Features/Tools/SwiftUI/FavoritingToolBannerView/FavoritingToolBannerView.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  FavoritingToolBannerView.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 4/20/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import SwiftUI
+
+struct FavoritingToolBannerView: View {
+    
+    // MARK: - Properties
+    
+    @ObservedObject var viewModel: FavoritingToolBannerViewModel
+    
+    // MARK: - Body
+    
+    var body: some View {
+        ZStack(alignment: .topTrailing) {
+            ColorPalette.banner.color
+            
+            HStack {
+                Spacer()
+                
+                Text(viewModel.message)
+                    .font(.system(size: 18))
+                    .multilineTextAlignment(.center)
+                    .foregroundColor(ColorPalette.gtGrey.color)
+                    .padding([.top, .bottom], 30)
+                    .padding([.leading, .trailing], 45)
+                
+                Spacer()
+            }
+            
+            Image("nav_item_close")
+                .padding(.top, 14)
+                .padding(.trailing, 18)
+                .onTapGesture {
```

Comment:
I'm curious about the size of the hit area with using a tapGesture directly on an Image, if it might be too small for this particular image.  I know in accessibility they mention a minimum size of 44x44 points for a tap area.  Something that might be worth looking into.

Reply:
good to know!  went ahead and made the frame 44x44 and adjusted the padding, so that makes the tappable area extend further to the left and bottom of the image